### PR TITLE
DOCSP-44478-hardware-specs

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION 
- Add new FAQ entry regarding hardware requirements for destination clusters.
- Add note on destination cluster disk space requirements in the Quickstart and oplog pages.

## STAGING 
- [FAQ](https://deploy-preview-488--docs-cluster-to-cluster-sync.netlify.app/faq/#what-hardware-specifications-should-the-destination-cluster-have-)
- [oplog Sizing Considerations section](https://deploy-preview-488--docs-cluster-to-cluster-sync.netlify.app/reference/oplog-sizing/#considerations)
- [Quickstart](https://deploy-preview-488--docs-cluster-to-cluster-sync.netlify.app/quickstart/#define-a-source-and-a-destination-cluster)

## JIRA
https://jira.mongodb.org/browse/DOCSP-44478

### CODE REVIEW

File: source/quickstart.txt [hunk 1]
```diff
@@ -80,6 +80,10 @@ Setup
       The number of nodes in the destination replica set does not have
       to equal the number of nodes in the source replica set.
 
+      .. important::
```
Comment (ajhuh-mdb) [inline source/quickstart.txt:83]: for internal reviewer: Do you think there's a better way to format/include this information in this section? Product specified that it'd be best here, but the way that the copy flows seems a bit clunky.

File: source/faq.txt [hunk 2]
```diff
@@ -47,6 +47,28 @@ have to run on the servers that host your MongoDB instances. When
...
+
+- CPU
+- Memory
+- Disk I/O 
+
+These hardware specifications ensure that the destination cluster can handle 
+``mongosync`` writes and that the sync can keep up with the source cluster's 
+workload.
+
+Additionally, the destination cluster must have enough disk storage to
```
Comment (jmd-mongo) [inline source/faq.txt:64]: [suggestion]

Consider using `includes/fact-oplog-disk-storage.rst` here. I think the only difference would be the loss of "Additionally".

File: source/quickstart.txt (hunk 1 above)
Comment (jmd-mongo) [reply source/quickstart.txt:83]: I understand your concern, but I can't think of a better way to present in. The copy definitely is important and that placement makes sense to me even if adds some visual noise due to multiple admonitions in the same step. I say go with it. Thanks!

**Review by jmd-mongo — COMMENTED** [review]
Thanks for this issue, @ajhuh-mdb! I left a suggestion and responded to your note re the admonition.

Thanks,
Joe

File: source/quickstart.txt (hunk 1 above)
Comment (ajhuh-mdb) [reply source/quickstart.txt:83]: Gotcha, thanks!

File: source/faq.txt (hunk 2 above)
Comment (ajhuh-mdb) [reply source/faq.txt:64]: Noted!

**Comment by ajhuh-mdb** [comment]
Thanks for the initial review @jmd-mongo ! This should be ready for another pass.

**Review by jmd-mongo — APPROVED** [review]
Thanks for this updates, @ajhuh-mdb! This one LGTM at this point.

Thanks,
Joe

**Comment by ajhuh-mdb** [comment]
Hi @edobranov 👋 Whenever you get the chance, could you review this docs PR regarding hardware specifications for destination clusters? TIA!

**Review by edobranov — APPROVED** [review]
LGTM % a couple optional suggestions, thanks!

**Comment by ajhuh-mdb** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|v1.9|[](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/494)|
|✅|v1.8|[](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/495)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

**Pull request merged**